amplipi/updater/asgi.py

```diff
@@ -59,10 +62,42 @@ class ReleaseInfo(BaseModel):
 dir_path = os.path.dirname(real_path)
 app.mount("/static", StaticFiles(directory=f"{dir_path}/static"), name="static")
 
-HOME = f"{os.environ.get('HOME')}/amplipi-dev" # standard install directory
+INSTALL_DIR = os.getenv('INSTALL_DIR', os.getcwd())
```

Comment:
This is one spot where I've changed the 'public interface' so to speak, but I think it functions. Prior, we took the `$HOME` var for a particular user and appended `/amplipi-dev`; simultaneously though, we've set the systemd service argument `WorkingDirectory` to the directory where AmpliPi is actually deployed, without making too many assumptions bout its name. This paradigm holds true as far back as commit 3a79c3c4eff994a254919be2d7930fca4e1213e5 titled "WIP web updates close to working", so for the straightforward vanilla case this ought to be fine; hackers may run into mild speedbumps with this though. In those cases, I've exposed an env var INSTALL_DIR (at least for the updater) while defaulting to the cwd.